Makefile

```diff
@@ -28,5 +28,9 @@ uninstall:
 	cd plugins/cwe_checker_type_inference; make uninstall; cd ../..
 	cd plugins/cwe_checker_type_inference_print; make uninstall; cd ../..
 
+documentation:
```

Comment:
make doc  ?

Reply:
make doc doesn't work. Or at least I haven't found a way to escape the `@doc` in the build command in a way such that it doesn't collide with make doc.

Reply:
ok, then leave it as it is